data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,132 @@
+                                user_location['latitude'])
+            org_location_dict[contrib.login] = location
+            logger.debug(f'{contrib.login} location {user_location}'
+                         f' added on map')
+    getorg.orgmap.output_html_cluster_map(org_location_dict,
+                                          folder_name=output_dir)
+
+    move_and_make_changes_in_files(output_dir)
+
+
+def move_and_make_changes_in_files(output_dir):
+    """
+    Move static files from 'output_dir' to django static folder and
+    the 'map.html' file to django templates directory to get it displayed
+    on the homepage with the needed django syntax and CSS in the html file.
+    :param output_dir: Directory from where the files have to be moved
+    :return: None
+    """
+
+    # Move leaflet_dist folder to static folder
+    leaflet_source_path = f'{os.getcwd()}/{output_dir}/leaflet_dist/'
+    leaflet_destination_path = f'{os.getcwd()}/static/leaflet_dist/'
+
+    # Remove existing leaflet_dir if exists
+    for root, dirs, files in os.walk(leaflet_destination_path):
+        for file in files:
+            os.remove(os.path.join(leaflet_destination_path, file))
+        os.rmdir(root)
+
+    os.renames(leaflet_source_path, leaflet_destination_path)
+
+    # Move org_locations.js to static folder
+    locations_source_path = f'{os.getcwd()}/{output_dir}/org-locations.js'
+    locations_destination_path = f'{os.getcwd()}/static/org-locations.js'
+    os.rename(locations_source_path, locations_destination_path)
+
+    # Make changes in map.html to support django syntax with needed CSS
+    with open('{}/map.html'.format(output_dir)) as f:
+        django_supported_htmls = []
+        for line in f:
+            line = line.strip()
+            if line.__contains__('<html>'):
+                django_supported_htmls.append('{% load staticfiles %}\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('</head>'):
+                adjust_prop = '''
+                <style>
+                    #map {
+                    width: 60%;
+                    height: 300px;
+                    margin: auto;
+                    box-shadow: 0px 0px 25px 2px;
+                    }
+                    @media only screen and (max-width:750px){
+                        #map {
+                            width: 90%
+                        }
+                    }
+                </style>\n
+                '''
+                meta_charset = '<meta charset="utf-8">'
+                adjust_prop = adjust_prop.strip().replace('    ', '')
+                django_supported_htmls.insert(6, meta_charset + '\n')
+                django_supported_htmls.append(adjust_prop + '\n')
+                django_supported_htmls.append(line + '\n')
+            elif line.__contains__('https://'):
+                line = line.replace('https:', '').replace(' />', '>')
```

Comment:
`line` is looping variable, don't use the same variable name to store the line after the modfication(use `newline` maybe). It might work in this case, but it's not the correct way to do it, it will not behave expectedly always.

Do change it everywhere you're doing this.